evm_loader/cli/src/main.rs

```diff
@@ -1193,6 +1196,77 @@ fn command_neon_elf(
     });
 }
 
+fn command_neon_elf(
+    config: &Config,
+    program_location: Option<&str>,
+) -> CommandResult {
+    if let Some(program_location) = program_location {
+        // Reading program data from file
+        let program_data = read_program_data(program_location).unwrap();
+        let program_data = &program_data[..];
+        read_elf_parameters(config, program_data);
+        Ok(())
```

Comment:
In other case you can also get rid of a following extra indentation just with returning Ok from here.

Reply:
Fixed